### PR TITLE
Publish deb package to a public APT repo

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +13,19 @@ this repository:
...
+echo 'deb https://us-central1-apt.pkg.dev/projects/radicle-services radicle-tools main' | sudo tee -a  /etc/apt/sources.list.d/artifact-registry.list
+sudo apt update
+sudo apt install radicle-tools
+```
+
+### From Homebrew on macOS
+
+```
+brew tap radicle-dev/radicle-client-tools https://github.com/radicle-dev/radicle-client-tools.git
+brew install radicle-dev/radicle-client-tools/radicle-tools
```
Comment (cloudhead) [inline README.md:30]: Can we (1) use our seed node directly, and (b) git rid of this `radicle-dev` prefix? maybe just `radicle`?

Comment (adaszko) [reply README.md]: yep, done

Comment (cloudhead) [reply README.md]: Cool, you forgot the (1) though :D

Comment (adaszko) [reply README.md]: Ugh, I didn't push it somehow 😬

**Pull request merged**